pkgs/by-name/ni/nixpkgs-vet/package.nix

```
@@ -0,0 +1,41 @@
+{
+  fetchFromGitHub,
+  lib,
+  nix,
+  nix-update-script,
+  rustPlatform,
+  versionCheckHook,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "nixpkgs-vet";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "NixOS";
+    repo = "nixpkgs-vet";
+    tag = finalAttrs.version;
+    hash = "sha256-J61eOTeDMHt9f1XmKVrEMAFUgwHGmMxDoSyY3v72QVY=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-H2JAIMJeVqp8xq75eLEBKiK2pBrgC7vgXXlqbrSUifE=";
+
+  nativeBuildInputs = [
+    versionCheckHook
+  ];
```

The `versionCheckHook` doesn't run when used like this

```suggestion
  nativeInstallCheckInputs = [
    versionCheckHook
  ];
  doInstallCheck = true;
```

---

To let you know: there is a little bit of docs about `versionCheckHook`: https://github.com/NixOS/nixpkgs/blob/master/doc/hooks/versionCheckHook.section.md

---

I figured it's a good idea to submit a little patch to upstream, so that `versionCheckHook` can be added in the next release anyways, because that way r-ryantm can test whether its updates are sound. https://github.com/NixOS/nixpkgs-vet/pull/158